tools/build.js

```diff
@@ -39,7 +42,7 @@ async function bundle(entryPoints, format, watch) {
         name: 'mark-runtime-modules-as-external',
         setup(pluginBuild) {
           pluginBuild.onResolve({ filter: /^\..*\.c?js$/ }, (args) => {
-            if (args.importer.includes(join('opennextjs-netlify', 'src'))) {
+            if (args.importer.includes(join(repoDirectory, 'src'))) {
```

Comment:
Main fix/adjustment - we were publishing runtime locally since move to Opennext organization and if locally repo clone directory was not named exactly `opennextjs-neltify`, then we would produce duplicates.